README.md

```diff
@@ -20,6 +20,17 @@ https://decred.org
 
 https://decred.org/downloads
 
+Core software:
+
+* dcrd: a Decred full node daemon (this)
+* [dcrwallet](https://github.com/decred/dcrwallet): a CLI Decred wallet daemon
+* [politeia](https://github.com/decred/politeia): a daemon for the Decred proposal system
```

Comment:
The user doesn't run the daemon, rather if they're doing things on the CLI, they only run the `politeiavoter` executable from the repository to interact with the remote service.

Reply:
Removed reference to politeia as part of the core software. Possibly, core software semantics would be better clarified in the main Decred docs.